Formula/kertish-dfs.rb

```diff
@@ -0,0 +1,30 @@
+class KertishDfs < Formula
+  desc "Kertish FileSystem and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dfs"
+  url "https://github.com/freakmaxi/kertish-dfs/archive/v21.2.0066.tar.gz"
+  sha256 "1ff300a803a0d3bd068e1492319a771912cccd4ba3a72098b6d05678813af966"
+  license "GPL-3.0-only"
+  head "https://github.com/freakmaxi/kertish-dfs.git"
+
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
```suggestion
  livecheck do
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
  end
```
Are you sure the livecheck block is needed? You're using a GitHub URL, so livecheck probably works without further specification.